tests/test_cli.py

```diff
@@ -625,6 +625,18 @@ def test_list_outputs_with_export(self):
         assert result.exit_code == 0
         assert result.output == "export SCEPTRE_Key='Value'\n"
 
+    @pytest.mark.parametrize("output_format,expected_output", [
+        ("yaml", '---\nmock-stack.yaml: mock-stack-external\n\n'),
+        ("text", '---\nmock-stack.yaml: mock-stack-external\n\n'),
+        ("json", '{\n    "mock-stack.yaml": "mock-stack-external"\n}\n')
+    ])
+    def test_list_stacks(self, output_format, expected_output):
+        result = self.runner.invoke(
+            cli, ["--output", output_format, "list", "stacks", "dev/vpc.yaml"]
```

Comment:
don't you just want to pass in `dev` instead of `dev/vpc.yaml` because `dev` is the stack group?